docs/en/5-templates-syntax.md

```diff
@@ -123,6 +123,29 @@ The template are applied. Result:
 <small class="item item_size_1"></small>
 ```
 
+If second argument of `mod()` was omited or equal is to '*' then templates with any
+value of modifier will be applied.
+
+```js
+block('a').mod('size').tag()('span');
+block('a').mod('size', '*').mix()('sized');
+```
+Both templates will be applied to BEMJSON entities if block equals to 'a' and
+'size' modifier exists (with value not `undefined` or `''`).
+
+```js
+{ block: 'a', mods: { size: 's' } },
+{ block: 'a', mods: { size: 10 } },
+{ block: 'a', mods: { size: '', theme: 'dark' } }
+```
+
+But templates will not be applied for entities:
```

Comment:
Guess we should not use entities here. BEMJSON nodes or Nodes?

`be applied for entities:` → `be applied to:`?